docs/commands/GetExAbsCurrentMC.md

```diff
@@ -0,0 +1,82 @@
+# GetExAbsCurrentMC
+
+Command to request/receive current consumption absolute values from device channels.
+
+This command can be sent periodically if [device report data parameter](../parameter-types.md#reporting-data-type) set to
+[current](../parameter-types.md#data-type) i.e. = `2`.
+
+
+## Request
+
+### Format
+
+| Size | Type   | Field               |
+| ---- | ------ | ------------------- |
+| `1`  | `byte` | extra flag = `0x1f` |
```

Comment:
extra or extended header flag? just ask for make it identically everywhere

Reply:
it's `extra flag` everywhere

but we can discuss a new name